WordPress/Classes/ViewRelated/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugViewModel.swift

```diff
@@ -16,40 +16,51 @@ final class BooleanUserDefaultsDebugViewModel: ObservableObject {
         }
     }
 
-    @Published var userDefaultsSections: Sections = []
+    @MainActor @Published private(set) var userDefaultsSections: Sections = []
 
-    init(coreDataStack: CoreDataStack = ContextManager.shared,
+    init(coreDataStack: CoreDataStackSwift = ContextManager.shared,
          persistentRepository: UserPersistentRepository = UserPersistentStoreFactory.instance()) {
         self.coreDataStack = coreDataStack
         self.persistentRepository = persistentRepository
     }
 
     func load() {
+        Task {
+            await self.load()
+        }
+    }
```

Comment:
This `load` method is for convenience only so that callers don't have to do the `Task { }` wrapping.